src/databases/config/connection.ts

```diff
@@ -1,18 +1,18 @@
-import { Sequelize } from "sequelize";
```

Comment:
What is the use of connection.ts file ?

Reply:
Ooh, that file was there before because I used Fabric's codebase. I forgot to remove it, but we don't need it.